### PR TITLE
Early result at passing empty values to BaseInflector methods

### PR DESCRIPTION
**Pull request by WinterSilence**
"foolproof"

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #19310

### CODE REVIEW

**Review by samdark — APPROVED** [review]
Please add a line for CHANGELOG.

**Review by Arhell — APPROVED** [review]

File: framework/CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Yii Framework 2 Change Log
 - Bug #19329: Fix `yii\web\GroupUrlRule` to properly normalize prefix (bizley)
 - Bug #19328: Passing null to parameter #1 ($string) of type string is deprecated in `yii\db\oci\Schema` (Arkeins)
 - Bug #19237: Fix OCI PHP 8.1 passing `null` to trim() (longthanhtran)
+- Enh #19312: Early result at passing empty values to `yii\helpers\BaseInflector` methods (WinterSilence)
```
Comment (samdark) [inline framework/CHANGELOG.md:22]:
```suggestion
- Bug #19312: Fix PHP 8.1 error when passing empty values to `yii\helpers\BaseInflector`  (WinterSilence)
```

**Comment by samdark** [comment]
Thanks.

**Comment by WinterSilence** [comment]
@samdark again repeat: **DONT CHANGE MY PRs WITHOUT MY CONSENT**. This is not `Fix PHP 8.1 error when passing null`.

**Comment by samdark** [comment]
It is.

**Comment by WinterSilence** [comment]
@samdark yeah, head need you only to eat?

**Comment by bizley** [comment]
> @samdark yeah, head need you only to eat?

Excuse me, maybe I'm reading this wrong because of your broken English but it looks like you are trying to insult Samdark. If not, my apologies, please state more clearly what you want to say. But if so... you better apologize right now.

**Pull request merged**